controller.py

```diff
@@ -34,6 +34,8 @@ def __init__(self):
         self.gridpacks_to_create_requests = []
         self.repository_tick_pause = 60
         self.tick_lock = Lock()
+        self.job_cores = [4, 8, 16, 32, 64]
```

Comment:
Pohweg should run  single core. How will we get a single one?

Reply:
Thanks for the feedback. I have included more options (1 and 2 cores), and now it is possible to set 1 core for the Pohweg Gridpacks.